apps/metadata-editor/src/app/app.module.ts

```diff
@@ -42,7 +52,19 @@ import { SignInPageComponent } from './sign-in/sign-in-page.component'
     TranslateModule.forRoot(TRANSLATE_DEFAULT_CONFIG),
     MatProgressSpinnerModule,
   ],
-  providers: [],
+  providers: [
```

Comment:
I think you just have to import the `FeatureCatalogModule` here, and it should take care of the rest